cli/src/semgrep/app/scans.py

```diff
@@ -301,7 +304,7 @@ def report_findings(
                 "errors": [error.to_dict() for error in errors],
                 "total_time": total_time,
                 "unsupported_exts": dict(ignored_ext_freqs),
-                "lockfile_scan_info": lockfile_scan_info,
+                "lockfile_scan_info": dependency_counts,
```

Comment:
could you add a snapshot test that snapshots the contents of a scan with dependency_counts in it?

Reply:
Tests exist! They just didn't get updated cause this PR doesnt change anything

Reply:
oh silly me